README.md

```diff
@@ -11,15 +11,16 @@
 
 A link aggregator / reddit clone for the fediverse.
 
-[Lemmy Dev instance](https://dev.lemmy.ml) *for testing purposes only*
+[Lemmy Dev instance](https://dev.lemmy.ml) _for testing purposes only_
```

Comment:
I prefer the * instead of _

Reply:
Sure, I'll revert them back to the old ones. Just my formatter being overly aggressive.